src/Makefile

```diff
@@ -21,15 +21,16 @@ lint:
 	-deadcode
 
 test:
-	go test -v -coverprofile=c.out ./...
-	@echo "go tool cover -func=c.out | sed 's/^github.com\/folio-org\/mod-reporting\/src\///'"
+	go test -v -coverprofile=coverage.out ./...
+	go test -json -coverprofile=coverage.out ./... > coverage.json
```

Comment:
It's a shame we have to run the tests twice to get the two different formats, but they've very fast so it's not a big deal in practice.